source/javascripts/components/WorkflowConfigPanel/components/EnvVarsCard.tsx

```diff
@@ -62,6 +62,9 @@ const EnvVarCard = ({ id, index, onRemove }: { id: string; index: number; onRemo
     transform: CSS.Translate.toString(transform),
   };
 
+  // NOTE: Default value isn't apply because the ref connected to the FormControl instead the Checkbox component in Bitkit
```

Comment:
typo
```suggestion
  // NOTE: Default value doesn't apply because the ref connected to the FormControl instead the Checkbox component in Bitkit
```

Reply:
thx 😅